dlp/api/Snippets.Tests/SendDataToTheHybridJobTriggerTests.cs

```diff
@@ -0,0 +1,93 @@
+using Xunit;
+using System.IO;
+using Google.Api.Gax.ResourceNames;
+
+namespace GoogleCloudSamples
+{
+    public class SendDataToTheHybridJobTriggerTests : IClassFixture<DlpTestFixture>
+    {
+        private DlpTestFixture _fixture;
+        public SendDataToTheHybridJobTriggerTests(DlpTestFixture fixture) => _fixture = fixture;
+
+        [Fact]
+        public void TestHybridJob()
+        {
+            var dlp = DlpServiceClient.Create();
+
+            // Create the Job Trigger.
+            var request = new CreateJobTriggerRequest
+            {
+                ParentAsLocationName = new LocationName(_fixture.ProjectId, "global"),
+                JobTrigger = new JobTrigger
+                {
+                    Triggers =
+                    {
+                        new JobTrigger.Types.Trigger
+                        {
+                            Manual = new Manual()
+                        }
+                    },
+                    InspectJob = new InspectJobConfig
+                    {
+                        Actions =
+                        {
+                            new Action
+                            {
+                                JobNotificationEmails = new Action.Types.JobNotificationEmails(),
+                                PublishToStackdriver = new Action.Types.PublishToStackdriver(),
+                            }
+                        },
+                        InspectConfig = new InspectConfig
+                        {
+                            InfoTypes =
+                            {
+                                new InfoType[]
+                                {
+                                    new InfoType { Name = "EMAIL_ADDRESS" }
+                                }
+                            },
+                            MinLikelihood = Likelihood.Possible,
+                            IncludeQuote = true,
+                        },
+                        StorageConfig = new StorageConfig
+                        {
+                            HybridOptions = new HybridOptions
+                            {
+                                Description = "Hybrid job trigger for data from the comments field of a table that contains customer appointment bookings",
+                            }
+                        }
+                    }
+                }
+            };
+            var hybridJobTrigger = dlp.CreateJobTrigger(request);
+
+            var text = "My email is test@example.org and name is Ariel.";
+
+            DlpJob result = SendDataToTheHybridJobTrigger.SendToHybridJobTrigger(_fixture.ProjectId, hybridJobTrigger.JobTriggerName.JobTriggerId, text);
+
+            // Delete the created job trigger.
+            dlp.DeleteJobTrigger(new DeleteJobTriggerRequest
+            {
+                Name = hybridJobTrigger.Name
+            });
+
+            // Delete the activated job.
+            JobsDelete.DeleteJob(result.Name);
```

Comment:
I think we should we invert the order. Delete the job and then delete the job trigger.

Reply:
Done